bower.json

```diff
@@ -14,6 +14,7 @@
     "angular-route": "1.3.x",
     "angular-mocks": "1.3.x",
     "angularytics": "^0.3.0",
-    "angular-messages": "~1.3.8"
+    "angular-messages": "~1.3.8",
+    "jquery": "~2.1.x"
```

Comment:
1) Since the `/material/bower.json` dependencies are copied to `/bower-material/bower.json`, we cannot include the jQuery as a dependency here; since we only want to install jQuery for `/angular/material/` testing purposes.

2) Similar to (1), we do not have a dependency upon angular-messages at deploy time; only for testing purposes.\

> Perhaps we need a solution as discussed for [Bower-Ignore-Dependencies](https://github.com/bower/bower/pull/1394)